src/utils.c

```diff
@@ -355,3 +355,15 @@ SEXP coerceUtf8IfNeeded(SEXP x) {
   UNPROTECT(1);
   return(ans);
 }
+
+//const void *DATAPTR_RO(SEXP x) {
```

Comment:
good catch about backporting that...
```
/builds/jangorecki/data.table/bus/test-310-cran-lin/data.table.Rcheck/data.table/libs/datatable.so: undefined symbol: DATAPTR_RO
```

Reply:
IIUC was added in roughly Sep '17 ~ 3.4

https://github.com/wch/r-source/commit/447cdb613a16c6419b6b9cc6bcfa9d9554f97db2

Reply:
To double-check, I grabbed the released tar.gz for R 3.4.4 and 3.5.0.  DATAPTR_RO was new in 3.5.0.  It's not always clear from the R-devel commits what is 'backported' to the next minor release and what is intended for the next major release.  So although it was committed during the R 3.4 series,  it wasn't released until R 3.5.0.